### PR TITLE
[feature-coule] add ttl for the workflow

### PR DESCRIPTION
**Pull request by merlintang**
This pr comes to fix the ttl of workflow as [issue](https://github.com/sql-machine-learning/sqlflow/issues/1824)

### CODE REVIEW

File: python/couler/couler/argo.py [hunk 1]
```diff
@@ -514,6 +516,10 @@ def yaml():
     ts.extend(_templates.values())
 
     wf["spec"] = {"entrypoint": entrypoint, "templates": ts}
+
+    if _workflow_ttl_cleaned is not None:
```
Comment (Yancey0623) [inline python/couler/couler/argo.py:520]: Just confused about how to config `_workflow_ttl_cleaned ` with `couler` command-line tool?

Comment (merlintang) [reply python/couler/couler/argo.py:520]: command line, we do not have this. you can configure this in your workflow code.

Comment (Yancey0623) [reply python/couler/couler/argo.py:520]: I see.

**Review by Yancey0623 — DISMISSED** [review]
LGTM

**Comment by Yancey0623** [comment]
I will add an optional parameter `SQLFLOW_WORKFLOW_TTL_SECONDS` in the SQLFlow server in the next PR.

**Comment by Yancey0623** [comment]
Hi @merlintang , the CI job failed on :

``` text

--- a/python/couler/couler/argo.py
+++ b/python/couler/couler/argo.py
@@ -519,7 +519,7 @@ def yaml():
 
     if _workflow_ttl_cleaned is not None:
         wf["spec"]["ttlSecondsAfterFinished"] = _workflow_ttl_cleaned
-    
+
     return wf
 
```

Maybe you can run `pre-commit` before committing the code.

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**